certbot/display/util.py

```diff
@@ -179,9 +179,12 @@ def checklist(self, message, tags, default_status=True, **unused_kwargs):
             self._print_menu(message, tags)
 
             code, ans = self.input("Select the appropriate numbers separated "
-                                   "by commas and/or spaces")
+                                   "by commas and/or spaces, or press enter to "
+                                   "select all domains shown")
 
             if code == OK:
+                if len(ans) == 0:
```

Comment:
nit: It might be nice to change this to:
```
if len(ans.strip()) == 0:
```
so the method still works if the user accidentally provides whitespace.